src/pages/play.tsx

```diff
@@ -1,13 +1,73 @@
-import React from 'react';
+import { navigate, StaticQuery } from 'gatsby';
+import React, { FunctionComponent, useState } from 'react';
 
 import { Deck } from '../components/Deck/Deck';
+import { Footer } from '../components/Footer/Footer';
 import { Header } from '../components/Header/Header';
-import Layout from '../components/Layout/Layout';
+import { Layout } from '../components/Layout/Layout';
+import { Wave } from '../components/Wave/Wave';
+import { dataQuery } from '../data/data';
+import { Answer, DataItem } from '../data/data.model';
+import buildDataset from '../data/utils/buildDataset';
+
+const goodAnswers = 5;
+const badAnswers = 5;
```

Comment:
Rules can be found here: https://sapient-france.slack.com/archives/C01KJHD86P2/p1611155408132000